core/src/test/java/org/elasticsearch/search/aggregations/bucket/FilterIT.java

```diff
@@ -118,10 +119,12 @@ public void testEmptyFilterDeclarations() throws Exception {
     }
 
     public void testWithSubAggregation() throws Exception {
+        Factory avgAgg = avg("avg_value");
+        avgAgg.field("value");
```

Comment:
maybe the Aggregation.Factory API should support chaining to be more in line with the rest of the API?